README.md

```diff
@@ -1,126 +1,114 @@
 - `agent5_repo_uri` and `agent6_repo_uri` become `agent_repo_uri` for all Agent versions.
 - `conf_dir` and `conf6_dir` become `conf_dir` for all Agent versions.
-- The repository file created on Linux is now named `datadog` for all agent versions instead of `datadog5`/`datadog6`.
+- The repository file created on Linux is now named `datadog` for all Agent versions instead of `datadog5`/`datadog6`.
 
-Usage
------
+### Configuration
 
 Once the `datadog_agent` module is installed on your `puppetserver`/`puppetmaster` (or on a masterless host), follow these configuration steps:
 
-1. Find your Datadog [API key][3].
+1. Obtain your [Datadog API key][2].
+2. Specify the module to install the Datadog Agent on your nodes.
 
-2. Specify the module on any nodes you wish to install the Datadog Agent.
-
-    ```
-    include datadog_agent
-    ```
+   ```conf
+   include datadog_agent
+   ```
 
     Or assign this module using the Puppet style Parameterized class:
 
-    ```
-    class { 'datadog_agent':
-        api_key => "<YOUR_DD_API_KEY>",
-    }
-    ```
+   ```conf
+   class { 'datadog_agent':
+       api_key => "<YOUR_DD_API_KEY>",
+   }
+   ```
 
-    On your `puppetserver`, enable reporting:
+3. On your `puppetserver`, enable reporting:
 
-    ```
-    class { 'datadog_agent':
-        api_key            => "<YOUR_DD_API_KEY>",
-        puppet_run_reports => true,
-    }
-    ```
+   ```conf
+   class { 'datadog_agent':
+       api_key            => "<YOUR_DD_API_KEY>",
+       puppet_run_reports => true,
+   }
+   ```
 
-    - To support reporting, your Puppet master needs the [dogapi][4] gem installed. To install, either run the Puppet Agent on your master with this configuration or install it manually with `gem`. You might need to restart your `puppetserver` service for the freshly installed `dogapi` gem to be picked up.
-    - `puppetserver_gem` is defined as a module dependency, it is installed automatically when the module is installed.
+    - To support reporting, your Puppet master needs the [dogapi][3] gem installed by running the Puppet Agent on your master with this configuration or installing it manually with `gem`. You may need to restart your `puppetserver` service after installing the `dogapi` gem.
+    - `puppetserver_gem` is defined as a module dependency. It is installed automatically when the module is installed.
 
-3. Include any other integrations you want the agent to use, for example:
+4. (Optional) Include integrations to use with the Agent, for example:
 
-    ```
-    include 'datadog_agent::integrations::mongo'
-    ```
+   ```conf
+   include 'datadog_agent::integrations::mongo'
+   ```
 
-    Some integrations do not come as a dedicated class. To install one of them, add its configuration in the manifest. Below is an example for the `ntp` check:
+    Add the integration configuration to the manifest if it does not have a [manifest with a dedicated class][6]. Below is an example for the `ntp` check:
```

Comment:
```suggestion
    In case an integration does not have a [manifest with a dedicated class][6], you can still add configuration for it. Below is an example for the `ntp` check:
```
Configuring an integration this way should be an exception. We try to provide dedicated classes for most integrations.